### DISCUSSION TITLE
Easy Tune Menu - Probe Z Offset Question

### DISCUSSION THREAD

**Discussion by csanii123** [mriscoc/Ender3V2S1#472] [Q&A] [answered]
I have the bed on my E3v2 leveled very well. In fact so well (and so close) that my prints are hard to get off the bed even when bed fully cooled.  I thus want to use the Z offset to move the nozzle away from the bed for the print. When I select Print and my gcode and then select Tune I can see the "Probe Z Offset" setting. Right now it is at 0. Does positive move nozzle away from bed or does positive move the nozzle closer to the bed ? I have looked online but I can't seem to locate the answer on what it does for the mrisoc firmware.  I have found conflicting answers for other firmware/slicers and given that I am already too close to the bed I don't want to make a mistake and damage my print bed by adjusting the wrong way.

So is positive setting in Tune Menu -> Probe Z Offset mean the nozzle moves closer or farther from print bed ?

**Comment by Triffen** [comment, marked as answer]
Positive setting lifts the nozzle above the bed. The Z stop is zero, below it are negative values, above it there are positive values.

**Reply by csanii123** [reply]
Thank you!
